messaging/src/main/java/org/axonframework/eventhandling/tokenstore/TokenStore.java

```diff
@@ -207,6 +207,19 @@ default boolean requiresExplicitSegmentInitialization() {
      */
     int[] fetchSegments(String processorName);
 
+    /**
+     * Returns an array of known {@code segments} for a given {@code processorName}.
+     * <p>
+     * The segments returned are segments for which a token has been stored previously and have not been claimed by another processor. When the {@link
+     * TokenStore} is empty, an empty array is returned.
+     *
+     * @param processorName The process name for which to fetch the segments
+     * @return an array of segment identifiers for the specified {@code processorName}, or an empty Optional if this method has not been implemented.
```

Comment:
```suggestion
     * Returns an array of known available {@code segments} for a given {@code processorName}.
     * <p>
     * The segments returned are segments for which a token has been stored previously and have not been claimed by another processor. When the {@link
     * TokenStore} is empty, an empty array is returned.
     *
     * @param processorName the processor's name for which to fetch the segments
     * @return an array of available segment identifiers for the specified {@code processorName}, or an empty {@code Optional} if this method has not been implemented.
```

Reply:
I'm in doubt whether to suggest an adjustment in the JavaDoc that clarifies further what "available" means. What do you think, @Vermorkentech?